packages/collector/test/test_util/ProcessControls.js

```diff
@@ -286,6 +286,11 @@ class ProcessControls {
     }
     return new Promise(resolve => {
       this.process.once('exit', resolve);
+      this.process.once('exit', () => {
+        this.process.pid = null;
+        resolve();
+      });
+
```

Comment:
@kirrg001 did you mean to delete line 288? Right now we would call `resolve` twice.

Reply:
Whoops 😅